proto/lib/livebook_proto/agent_connected.pb.ex

```diff
@@ -0,0 +1,10 @@
+defmodule LivebookProto.AgentConnected do
+  use Protobuf, syntax: :proto3, protoc_gen_elixir_version: "0.12.0"
+
+  field :id, 1, type: :int32
+  field :name, 2, type: :string
+  field :public_key, 3, type: :string, json_name: "publicKey"
+  field :deployment_group, 4, type: LivebookProto.DeploymentGroup, json_name: "deploymentGroup"
+  field :secrets, 5, repeated: true, type: LivebookProto.Secret
+  field :file_systems, 6, repeated: true, type: LivebookProto.FileSystem, json_name: "fileSystems"
```

Comment:
Why do we need the json names? I thought Elixir on the server would always use `deployment_group: ...` as keys?

Reply:
It's autogenerated for Protobuf, he uses the Java pattern (`camelCase`) for keys, but Elixir converts back to `snake_case`.